File: 004/004.py
import triangle

a, b, c = map(float, input("Nhap 3 canh tam giac: ").split())

if triangle.check(a, b, c) == True:

    print(triangle.type(a, b, c))

    print(f"Dien tich S = {triangle.Heron(a, b, c)}")
else:
    print("Khong phai tam giac")

File: 004/004.rb
require_relative "triangle"

print "Nhap 3 canh tam giac: "
a, b, c = gets.split.map(&:to_f)

if check(a, b, c)

  puts "#{type(a, b, c)}"
  puts "Dien tich S = #{Heron(a, b, c).round(1)}"
else
  puts "Khong phai tam giac"

end
